From 9c3dab5fa11578c6780fddb11f1d2b7225defe38 Mon Sep 17 00:00:00 2001
From: Mingcong Bai <jeffbai@aosc.io>
Date: Mon, 9 Dec 2024 18:04:57 +0800
Subject: [PATCH 225/301] AOSCOS: hwmon: nct7511: revise .probe() in struct
 i2c_driver

Since commit 03c835f498b5 ("i2c: Switch .probe() to not take an id
parameter"), the `.probe()' member function only takes an i2c_client as
its sole parameter. Remove the second `i2c_device_id' parameter.

Signed-off-by: Mingcong Bai <jeffbai@aosc.io>

Signed-off-by: Kexy Biscuit <kexybiscuit@aosc.io>
---
 drivers/hwmon/nct7511.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/drivers/hwmon/nct7511.c b/drivers/hwmon/nct7511.c
index e5006ecd8346..808755e2c37d 100644
--- a/drivers/hwmon/nct7511.c
+++ b/drivers/hwmon/nct7511.c
@@ -744,8 +744,7 @@ static int nct7511_init_chip(struct nct7511_data *data)
 	return regmap_update_bits(data->regmap, REG_VMON_ENABLE, 0x03, 0x03);
 }
 
-static int nct7511_probe(struct i2c_client *client,
-			 const struct i2c_device_id *id)
+static int nct7511_probe(struct i2c_client *client)
 {
 	struct device *dev = &client->dev;
 	struct nct7511_data *data;
-- 
2.48.1

